### ISSUE TITLE
[suggestion] 1 top-level bullet point for each person

### ISSUE THREAD

**Issue by joon-ko** [jynnie/accountability#1]
obviously, it's likely that we have more than one 'goal.' 

i propose a suggestion so that each person has one top-level bullet point for their name, and then a person's goals is a second-level bullet point, like so:

* person1
    * goal1
        * deadline, goal1
        * progress, goal1
    * goal2
    * ...
* person2
    * goal1
    * goal2
* person3
    * ...

this would improve  ~~~~~ readability ~~~~~ so we can more easily poke our friendos when deadlines approach! :0

**Comment by jynnie** [comment]
hmm, i was thinking about organizing by goaled deadline - but you're probably right; if everyone dumps all their project here at once, we'll need a better organization system

**Comment by mysticuno** [comment]
Just reformatted to see what it would look like with Joon's suggestions, I think we should do a mixture of the two where the top hierarchy is as above, and then each person is responsible for ordering their goals chronologically (though this is also pretty free form so we can also just be p chill about this; it's more about the act of writing your goals down and making them visible so that other people can keep you accountable more so than the specific methodology itself imo)

**Comment by joon-ko** [comment]
yeah, personally i think the separation of top-level bullets by people is useful for people because they're going to want to group their goals in one block anyway so they can track and update them better, and support leaving anything else up to them?

i do like the reformat tho +1

**Comment by jynnie** [comment]
Hmm I find it a bit hard to read with all the nested bullet points though .-. Is there another way we could format this?

What do people feel about (not completely thought through - meh):
* person1
   * goal1 [deadline]
       * progress [date of update]
   * goal2

(also test test lets see if markdown renders this)

- [ ] checkbox?
- [x] checked?
